Wikipedia/Code/ExploreViewController.swift

```diff
@@ -320,7 +320,9 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
 
     private func setupFetchedResultsController() {
         let fetchRequest: NSFetchRequest<WMFContentGroup> = WMFContentGroup.fetchRequest()
-        fetchRequest.predicate = NSPredicate(format: "isVisible == YES && (placement == NULL || placement == %@)", "feed")
+        let now: Date = (Date() as NSDate).wmf_midnightUTCDateFromLocal
+        let oldestDate = now.addingTimeInterval(-(60 * 60 * 24 * Double(WMFExploreFeedMaximumNumberOfDays)))
+        fetchRequest.predicate = NSPredicate(format: "isVisible == YES && (placement == NULL || placement == %@) && contentMidnightUTCDate >= %@", "feed", oldestDate as NSDate)
```

Comment:
It looks like I'm only seeing top read items in the explore feed. Inspecting the sqlite db, it seems `contentMidnightUTCDate` is only being set on top read items. I could be misinterpreting something, but should we leverage `midnightUTCDate` here instead, which seems to be set for every element?

Reply:
Yikes, good catch! 😅 I changed to `midnightUTCDate`. I also inspected some of the libraries I've received and it seems to always be populated, so I think we're safe with that.